src/Symfony/Component/DomCrawler/Tests/CrawlerTest.php

```diff
@@ -27,6 +27,12 @@ public function testConstructor()
         $this->assertCount(1, $crawler, '__construct() takes a node as a first argument');
     }
 
+    public function testGetBaseHref()
+    {
+        $crawler = new Crawler(null,  null, 'http://symfony.com');
+        $this->assertEquals('http://symfony.com', $crawler->getBaseHref());
```

Comment:
You could put the string `'http://symfony.com'` into a single variable, so one of them can't possibly be typo'd.